pkgs/tools/typesetting/asciidoc/default.nix

```diff
@@ -249,11 +254,13 @@ stdenv.mkDerivation rec {
         -e "s|^XMLLINT =.*|XMLLINT = '${libxml2.bin}/bin/xmllint'|" \
         -e "s|^EPUBCHECK =.*|EPUBCHECK = 'nixpkgs_is_missing_epubcheck'|" \
         -i a2x.py
-  '' + ''
-    for n in $(find "$out" . -name \*.py); do
-      sed -i -e "s,^#![[:space:]]*.*/bin/env python,#!${python2}/bin/python,g" "$n"
-      chmod +x "$n"
-    done
+  '' else ''
+    sed -e "s|^ENV =.*|ENV = dict(XML_CATALOG_FILES='${docbook_xml_dtd_45}/xml/dtd/docbook/catalog.xml ${docbook_xsl_ns}/xml/xsl/docbook/catalog.xml ${docbook_xsl}/xml/xsl/docbook/catalog.xml')|" \
+        -e "s|^XSLTPROC =.*|XSLTPROC = '${libxslt.bin}/bin/xsltproc'|" \
+        -e "s|^XMLLINT =.*|XMLLINT = '${libxml2.bin}/bin/xmllint'|" \
```

Comment:
From looking at the a2x.py it just wants these in PATH. So I think also, these programs being in nativeBuildInputs should be enough?

Reply:
@dasJ same as https://github.com/NixOS/nixpkgs/pull/102398/files#r552481684

Reply:
@zimbatm https://github.com/NixOS/nixpkgs/pull/102398#issuecomment-725381663

While I don't really know enough about asciidoc to judge this myself, I think we cannot just put them into `nativeBuildInputs` since that would require putting them into the `nativeBuildInputs` of every build that uses a2x